pkg/cli/webhook/webhook.go

```diff
@@ -102,3 +102,33 @@ func (w *Options) Install(ctx context.Context, providerType string) error {
 	// update repo cr with webhook secret
 	return w.updateRepositoryCR(ctx, response)
 }
+
+func GetProviderName(url string) (string, error) {
+	var (
+		err          error
+		providerName string
+	)
+	switch url {
+	case "github":
```

Comment:
is the url supposed to be just "github" or "gitlab"... ?

Reply:
Updated now

Yes URL is full and while committing i thought i changed but i dint :grin:

Reply:
it may not happen if you covered it with a unittest ;)

Reply:
Yes 
I did cover that scenario but  i dint run testcase after adding switch case because of that i missed that failure locally :laughing: